recipes/nnetsauce/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.4" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 3341147dff839728410703f59670858481a4520994d80fc1b2a821e9c52895da
+
+build:  
+    number: 0  
+    script: "{{ PYTHON }} -m pip install . -vv"
+    skip: true  # [py < 35]
+
+requirements:
+    build:    
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+    host:
+        - python
+        - pip
+        - setuptools 
+        - cython >=0.29.21
+        - numpy >=1.14.0
```

Comment:
It looks like you need `scipy` with the appropriate version restriction from your `setup.py` in the `host` section because of the custom checks.